configure.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+echo "Running configuration script"
+export DEBIAN_FRONTEND=noninteractive
+
+echo "US/Central" | sudo tee /etc/timezone
```

Comment:
Yeah!  Go, Chicago!  W000000t!

Reply:
Prior to opening the PR I was tempted to look up the format for east coast to replace Central but decided it would be petty ;)